packages/node_modules/@webex/redux-module-spaces/src/actions.js

```diff
@@ -178,17 +183,22 @@ export function updateSpaceWithActivity(activity, isSelf, isReadable = false) {
 export function fetchSpace(sparkInstance, space) {
   return (dispatch) => {
     dispatch(storeInitialSpace(space.id));
+    dispatch(createConversationBegin());
```

Comment:
Could you throw a comment as to why we are storing this in the conversation redux as well?

```js
// Until the conversation redux store is fully migrated, this will duplicate the data into it
```

Reply:
I've added a variation of this comment to help explain these details. Let me know if this is okay.